experiment/measurer/constants.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
```

Comment:
Please don't do this. I don't like constants files.
I'd rather just import it from run_coverage.

Reply:
Done